### ISSUE TITLE
An inaccessible audio URL can be added without any warning or feedback

### ISSUE THREAD

**Issue by ScenarioKG** [wordpress-mobile/WordPress-Android#21168]
### Expected behavior
The system should provide feedback or a notification indicating if the URL is incorrect or inaccessible, preventing further action until the issue is resolved.

### Actual behavior
In step 7, no feedback or notification is provided about the incorrectness of the URL, leaving users unaware of any issues.
In step 9, a '0:00 / 0:00' audio block is added.

### Steps to reproduce the behavior
1. Click the 'Create a post or page' icon
2. Click the 'Blog post' button
3. Click the 'Audio' icon
4. Click the 'Insert from URL'
5. Enter an inaccessible URL, e.g., "https://example.com/audiofile.mp3"
6. Tap the 'Done' button on the keyboard
7. Click the outside of the 'Insert from URL' pop-up
8. Click the 'Three-dot' menu
9. Click the 'Preview' button

##### Tested on [device], Android [version], JPAndroid / WPAndroid [version]
Galaxy A14 5G, Android 13, JPAndroid / WPAndroid 25.2.1 (1444)

https://github.com/user-attachments/assets/3ae616b8-4549-4e88-be32-33906f509890

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by WesleyJammer** [comment]
hey, i am pretty new with opensource and for a school project i have to work on an opensource project. i am curious if this issue is still open and if so, i can work on it?

**Comment by jkmassel** [comment]
This seems like an issue better solved in Gutenberg Core – it's not unique to WP Android.

WDYT @dcalhoun?

**Comment by dcalhoun** [comment]
Yes, I agree this should be addressed or improved in Gutenberg Core. 

~I'll note that GutenbergKit mirrors Gutenberg web in that it does display a small "error" text when this is situation occurs.~ The referenced "error" text is unrelated to GutenbergKit or Gutenberg web, it is simply Safari audio player functionality and it not present in Android Chrome. Regardless, improvements should still occur in Gutenberg core. 

<details><summary>GutenbergKit audio block error</summary>
<p>

![GutenbergKit displaying error message for failed audio block](https://github.com/user-attachments/assets/71fe2450-40d1-4690-afd5-c9f65de019fc)

</p>
</details>
